### ISSUE TITLE
Error parsing `<SHORTFNAME>` field

### ISSUE THREAD

**Issue by delphidabbler** [delphidabbler/vied#48]
https://github.com/delphidabbler/vied/blob/382fbc85fbf8efbec3658a326943dd3fc05d47e0/Source/UVInfo.pas#L780

All other fields that reference contents of a string info item evaluate fields in string info item rather than simply referencing it. For example, here's how `<PRODUCTNAME>` is parsed:

https://github.com/delphidabbler/vied/blob/382fbc85fbf8efbec3658a326943dd3fc05d47e0/Source/UVInfo.pas#L781

So, should `<SHORTFNAME>` be parsed like this instead?:

```pascal
case I of
  // ...
  tkSHORTFNAME: Result := RemoveExtension(EvaluateFields(siOriginalFileName);
  // ...
end;
```

**Comment by delphidabbler** [comment]
To test, create a `.vi` file with a field (say `<YEAR>`) referenced in the `OriginalFileName` string info item and reference `<SHORTFNAME>` in another string info item (say `Comments`).

Then generate a resource file and check that all fields have been parsed.

**Comment by delphidabbler** [comment]
> To test, create a `.vi` file with a field (say `<YEAR>`) referenced in the `OriginalFileName` string info item and reference `<SHORTFNAME>` in another string info item (say `Comments`).
> 
> Then generate a resource file and check that all fields have been parsed.

I created a new file and set

    Original File Name = myfile<YEAR>.exe
    Internal Name = Short:<SHORTFNAME>

And got the following:

```
FILEOS VOS__WINDOWS32
FILETYPE VFT_APP
{
 BLOCK "StringFileInfo"
 {
  BLOCK "080904B0"
  {
   VALUE "InternalName", "Short:myfile<YEAR>\000"
   VALUE "OriginalFileName", "myfile2023.exe\000"
  }
 }
 BLOCK "VarFileInfo"
 {
  VALUE "Translation", 0x0809, 1200
 }
}
```

Which confirms that `<SHORTFNAME>` is buggy.

**Comment by delphidabbler** [comment]
Fixed at commit 6ef4d91

Now the test .vi file works as expected:

```
Original File Name = myfile<YEAR>.exe
Internal Name = Short:<SHORTFNAME>
```

Now working:

```
FILEOS VOS__WINDOWS32
FILETYPE VFT_APP
{
 BLOCK "StringFileInfo"
 {
  BLOCK "080904B0"
  {
   VALUE "InternalName", "Short:myfile2023\000"
   VALUE "OriginalFileName", "myfile2023.exe\000"
  }
 }
 BLOCK "VarFileInfo"
 {
  VALUE "Translation", 0x0809, 1200
 }
}
```
